convergence.md

```diff
@@ -1,3 +1,499 @@
+
+Areas of potential disagreement should be identified *before* the merge so that compromise resolutions can be found either in advance or during the merge process. Changes to specific implementation details of existing features might be necessary but *as a general rule*, there should be no loss of functionality between the two projects. If something worked pre-merge in either Node.js or io.js, it should continue to work post-merge.
+
+### Merge Process
+
+Of the two existing repositories, `iojs/io.js` has been the most active since the fork. Since that time, there has been some cherry-picking of commits between the two projects but the activity has been inconsistent and the separate code bases have diverged in fairly significant ways. Based on the commit history, the easiest merge approach will be to create the new `nodejs-foundation/node` repository as a Fork of the existing `iojs/io.js` (`nodejs-foundation/io.js`) repository, then cherry pick commits from `joyent/node` (`nodejs-foundation/node`) to merge in. Unfortunately, this will not be as easy as simply using `git format-patch` and `git am`.
+
+The commits generally fall into three categories:
+
+* Features
+* Fixes
+* Doc & Test
+
+Feature commits will most likely need to be reimplemented using the original commit as a guide. There are some that may be able to transfer directly but they are likely rare.
+
+Fix commits are tricky because, in many cases, the issue may have already been fixed in io.js but in a different way than what was landed in joyent/node. For these, the process is:
+
+1. determine if there is an existing test case for the fix and if there is not, create one,
+2. test to see if the issue has already been fixed in the io.js base,
+3. if it has, compare the two fixes and determine if there were any additional pieces added to the `joyent/node` fix that need to be added
+4. if it has not been fixed, port the fix from `joyent/node`
+
+Doc & Test commits are handled in generally the same way as Fix commits.
+
+#### Merging from Parallel io.js and node.js tracks
+
+While the repositories are being merged, the existing io.js and node release streams will continue to operate. This implies that there will be new commits to both that will also need to be merged into the converged repo. Those are to be handled in generally the same way as the other commits.
+
+However, once the Foundation launches, the decision making process for both release streams follows the Foundation TSC guidelines with the combined TSC overseeing both individual release lines. Decisions about features / fixes going into the separate node.js and io.js lines will be the responsibility of the *combined* TSC. This helps to ensure that the decisions for what items make it into the converged project are being properly reviewed following the Foundation process.
```

Comment:
The "new" process is essentially a slightly revised version of the existing io.js process. The main point here is that the TCs are brought together and that decisions are made with convergence in mind. We don't want new conflicting things going independently into each release line which ends up just causing us more pain later. 

Plus, the ability to evolve the process is built into the process so if things don't work as well as they should, we iterate and improve them as opposed to abandoning back to the "old" process.